ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/AstProcessorApplication.java

```diff
@@ -0,0 +1,23 @@
+package edu.usb.argos.ASTProcessor;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.List;
+
+@SpringBootApplication
+public class AstProcessorApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(AstProcessorApplication.class, args);
+
+		String testFilePath = "src/main/java/edu/usb/argos/ASTProcessor/ExampleClass.java";
```

Comment:
That path not works on my project, but with this works normally `"ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/ExampleClass.java"`, I don't know if happen the same with you.

Reply:
Could you post an image of the problem you are having, please?